web/ui/src/Services/servicesFactory.js

```diff
@@ -29,7 +29,7 @@
         angular.extend(newFactory, {
             // TODO - update list by application instead
             // of all services ever?
-            update: function(force){
+            update: function(force, skipUpdateInstances){
```

Comment:
updating instances can be pretty costly, so allow manual skip. specifically for service map, we want to manually update the instances so we can be notified when the update completes.